<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>top_menu</class>
 <widget class="QWidget" name="top_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>766</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Слепопечать v0.1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#top_menu {
	background-color: transparent;
	background: none;
	border: none;
	background-repeat: none;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,5,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="record_logo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>650</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#record_logo {
background-color: transparent;
border-image: url(:/images/images/top_logo.png);
background: none;
border: none;
background-repeat: none;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,0">
       <item>
        <widget class="QPushButton" name="record_words_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#record_words_button {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 20pt;
}

#record_words_button:pressed {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 18pt;
}</string>
         </property>
         <property name="text">
          <string>Слова</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="record_nums_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#record_nums_button {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 20pt;
}

#record_nums_button:pressed {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 18pt;
}</string>
         </property>
         <property name="text">
          <string>Числа</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="record_letters_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#record_letters_button {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 20pt;
}

#record_letters_button:pressed {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 18pt;
}</string>
         </property>
         <property name="text">
          <string>Буквы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="record_customs_button">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#record_customs_button {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 20pt;
}

#record_customs_button:pressed {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 18pt;
}</string>
         </property>
         <property name="text">
          <string>Кастом</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="record_table">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="midLineWidth">
        <number>5</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::CopyAction</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="record_info_label">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Нажми на слово, чтобы отобразить рекорды категории</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="record_quit_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#record_quit_button {
background-color: transparent;
border-image: url(:/icons/images/widget_quit.png);
background: none;
border: none;
background-repeat: none;
}
#record_quit_button:pressed
{
   border-image: url(:/icons/images/widget_quit_clicked.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qrc_files/pictures.qrc"/>
 </resources>
 <connections/>
</ui>
